.travis/install.sh

```diff
@@ -11,12 +11,13 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
   conda config --set always_yes yes --set show_channel_urls true --set changeps1 no
   conda update -q conda
   conda config --add channels conda-forge
-  conda info -a
   conda init bash
-  conda env create -f environment.yml
+  # Force python 3.8
+  conda create -n abc-dev python=3.8 github3.py
```

Comment:
here i'm manually creating the envt. no need for a file right now because there is only one package being installed. this should work for now but it's time to revisit the requirements of this package and what goes in dev-requirements vs what is here installed via conda in all os'